modules/snooker/src/main/ts/ephox/snooker/resize/BarManager.ts

```diff
@@ -124,6 +124,9 @@ export const BarManager = (wire: ResizeWire): BarManager => {
   const findClosestEditableTable = (target: SugarElement<Node>): Optional<SugarElement<HTMLTableElement>> =>
     SelectorFind.closest<HTMLTableElement>(target, 'table', isRoot).filter(ContentEditable.isEditable);
 
+  const isResizer = (target: SugarElement<Element>): boolean =>
+    target.dom.classList.contains('ephox-snooker-resizer-bar') || target.dom.classList.contains('ephox-dragster-blocker');
```

Comment:
```suggestion
    Class.has(target, 'ephox-snooker-resizer-bar') || Class.has(target, 'ephox-dragster-blocker');
```

Or you could do `Classes.hasAny(target, ['ephox-snooker-resizer-bar', 'ephox-dragster-blocker'])` but that's probably not worth it for only two.